figma-shop/src/app/pages/products/products.component.html

```diff
@@ -0,0 +1,36 @@
+<div class="inner-product">
+    <div class="main-title">
+        <h1>All Products</h1>
+        <div class="count-products">
+            <span>117</span>
+            <span>Products</span>
+        </div>
+    </div>
+
+    <div class="sort-select">
+        <select name="" id="">
+            <option value="">asdasd</option>
+            <option value="">asdasd</option>
+            <option value="">asdasd</option>
+        </select>
+    </div>
+
+    <main class="main">
+        <div class="row">
+            <div class="filter col-md-3 col-lg-3 border">
+                
+            </div>
+            <div class="products col-md-9 col-lg-9 border">
+                <div class="product-card" *ngFor="let product of allProducts">
+                    <app-product-card [product]="product"></app-product-card>
+                </div>
+            </div>
+        </div>
+    </main>
+
+
```

Comment:
You have to accustom yourself to write "clean" code, especially when you are making prs. Look to every space and aligning. In current case you have too many spaces in html and also at the end of the file should be only one space!